### DISCUSSION TITLE
hide sidebar section when needed and automatic filling of the homepage with recently added notes/docs.

### DISCUSSION THREAD

**Discussion by msavdert** [imfing/hextra#120] [General]
Thank you for this wonderful template. It's really simple to install and edit even for someone like me who doesn't know much about this stuff. I just have a couple questions:

1) Is it possible to hide the Sidebar at any time and expand it at any time? like:

https://www.docsy.dev/docs/adding-content/navigation/#section-menu-options

2) Is it possible to publish a sub-article without showing it in the sidebar? like:
```markdown
---
title: Configuration
sidebar:
  open: false
---
```

This option only works for _index.md. This option didn't work for other .md files under the same folder to not show them in the sidebar.

and, Is it possible for the documents to fully fit (%100) across the entire page when I hide the sidebar?

3) Is it possible to 100% fit the whole page for the blog tab only when needed.

4) Can I edit the home page to automatically show the last 10 or 20 newly added notes? like recently added.

**Comment by imfing** [comment]
Hi, thanks for the questions.

> Is it possible to hide the Sidebar at any time and expand it at any time? like:
https://www.docsy.dev/docs/adding-content/navigation/#section-menu-options

it's not supported right now, but should be possible to add.

> Is it possible to publish a sub-article without showing it in the sidebar? like:

similar to above, i could add this feature.

> This option only works for _index.md. This option didn't work for other .md files under the same folder to not show them in the sidebar.

I will take a look at it.


> Is it possible to 100% fit the whole page for the blog tab only when needed.

Have you tried https://imfing.github.io/hextra/docs/guide/configuration/#page-width
do you mean extend the content to the full page width?

> Can I edit the home page to automatically show the last 10 or 20 newly added notes? like recently added.

I could also add a flag in the settings.

**Reply by PTLSP** [reply]
Author your Giscus parameters were recorded in the open source program so I accidentally used your repository I am now starting to learn how to replace it with my own I have troubled you and I feel very sorry about it

**Reply by imfing** [reply]
@PTLSP no worries, feel free to open a discussion if you ran into any problem.

**Comment by PTLSP** [comment]
test

**Comment by PTLSP** [comment]
www

**Reply by imfing** [reply]
@PTLSP please use your own repo for testing

**Reply by PTLSP** [reply]
Sorry this is my first time using 😥
